src/components/Pages/Designer.vue

```diff
@@ -197,6 +197,20 @@
 </template>
 
 <script>
+import GeogebraInterface from '../Geogebra/GeogebraInterface';
+
 export default {
+    mounted() {
+        // simple example code to show how to initialize GeoGebra
+        const GI = new GeogebraInterface('geogebraDesigner'); // initializing our custom class
```

Comment:
One blank life before entering new block.

Reply:
between export default and mounted() ?